index.d.ts

```diff
@@ -142,8 +160,8 @@ export declare class Request {
   rawHeaders?: {
     [key: string]: string | undefined;
   };
-  body: any;
-  rawBody: string;
+  body: Buffer | string | object | undefined;
```

Comment:
is this the known full set, or are you approximating to extend `any` to include `Buffer | string`?